src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -909,7 +909,13 @@ class RsErrorAnnotator : AnnotatorBase(), HighlightRangeExtension {
 
         val owner = attr.owner ?: return
 
-        val reprArgs = attr.metaItem.metaItemArgs?.metaItemList.orEmpty()
+        val metaItemArgs = attr.metaItem.metaItemArgs
+
+        if (metaItemArgs == null) {
+            RsDiagnostic.ReprNoParentheses(attr).addToHolder(holder)
```

Comment:
Let's annotate only the corresponding `attr.metaItem` to specify the error range more preciesely.
Also, it allows us to correctly use the same diagnostic for cases like `#[cfg_attr(unix, repr)]` in future to highlight only `repr` text